presto-docs/src/main/sphinx/sql/grant.rst

```diff
@@ -26,6 +26,10 @@ The optional ``WITH GRANT OPTION`` clause allows the grantee to grant these same
 
 For ``GRANT`` statement to succeed, the user executing it should possess the specified privileges as well as the ``GRANT OPTION`` for those privileges.
 
+.. note::
+
+    Ensure that authentication has been enabled before running any of the authorization commands.
```

Comment:
This note is intended for server administrators. It's purpose is to warn them about the scenario when commands such as `GRANT/REVOKE` are run without authentication. Since there is no way to prohibit administrators from running those commands, we thought the best we can do is to warn them.